ui/raidboss/data/06-ew/ultimate/dragonsongs_reprise_ultimate.ts

```diff
@@ -257,6 +491,85 @@ const triggerSet: TriggerSet<Data> = {
       netRegexKo: NetRegexes.startsUsing({ id: '63C6', source: '기사신 토르당', capture: false }),
       response: Responses.aoe(),
     },
+    {
+      id: 'DSR Sanctity of the Ward Direction',
+      type: 'Ability',
+      netRegex: NetRegexes.ability({ id: '63E1', source: 'King Thordan', capture: false }),
+      netRegexDe: NetRegexes.ability({ id: '63E1', source: 'Thordan', capture: false }),
+      netRegexFr: NetRegexes.ability({ id: '63E1', source: 'Roi Thordan', capture: false }),
+      netRegexJa: NetRegexes.ability({ id: '63E1', source: '騎神トールダン', capture: false }),
+      netRegexCn: NetRegexes.ability({ id: '63E1', source: '骑神托尔丹', capture: false }),
+      netRegexKo: NetRegexes.ability({ id: '63E1', source: '기사신 토르당', capture: false }),
+      condition: (data) => data.phase === 'thordan',
+      delaySeconds: 4.3,
+      promise: async (data) => {
+        // Only need to know one of the knights locations, Ser Janlenoux (3635)
```

Comment:
Can you add some comments about how this works? I kind of expected that you'd have to look at heading here and not position.

Reply:
The side the white knight, Ser Janlennoux, is on (relative to east/west) can be used to determine where both knights will dash.

Reply:
What if he lands on south or north? I've seen intercards and cards here (but I have no example of him on south or north, despite looking through a number of videos).

Reply:
Mostly I'm asking because we do calls based on the way the knight closest to the drk is facing.  e.g. if you're on the drk and looking mid the closest knight is looking left (either diagonal towards you or away from you) then you're going left/clockwise, and I think that would solve the north/south issue, unless there's some fixed "if it's north it's always left" sort of limited possibility?

Unless it's really the case that it can never be north or south??

Reply:
In all my pulls, I have not seen north or south be spawned, but that just means they spawned more towards the intercards than the other cases. I haven't exactly looked where the white knight can spawn.

Reply:
To add, it should call out unknown if it spawned north/south.

Reply:
some further testing, this only works for the 2 move strat. the 1 move strat will probably need heading?

Reply:
Ok, I looked at the fflogs data. The two gladiators spawn in one of two positions: West (95, 100) or East (105, 100). They only look in two directions as well. Ser Janlenoux only ever looks towards the north (northeast or northwest), and Ser Adelphel only looks towards the south (southwest or southeast). So I think if it calls out wrong I either have the wrong one of three Ser Janlenoux or the delaySeconds needs adjustment?

Reply:
I believe it's resolved now. Testing for if I need to increase delay more... but several pulls and it's been solid.
Could opt for changing cw/ccw to left/right and that way either 1-move or 2-move strats can use it without alterations (they just change direction they'd be looking), but that is more vague.